catalog/task/validate-single-component/0.1/validate-single-component.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: validate-single-component
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task validates that the snapshot only contains a 
+    single component. The task will fail otherwise.
```

Comment:
Can you copy this description into the README in place of the current on there? It reads better

Reply:
Done, thanks!